add-on/chrome/modules/LoopRooms.jsm

```diff
@@ -595,6 +596,24 @@ var LoopRoomsInternal = {
   },
 
   /**
+   * Request information about number of participants joined to a specific room from the server.
+   * Used to determine infobar message state on the number of participants in the room.
+   *
+   * @param {String}  roomToken Room identifier
+   * @return {Number} Count of participants in the room.
+   */
+  getNumParticipants: function(roomToken) {
+    MozLoopService.log.debug("bar roomToken", roomToken);
+    MozLoopService.log.debug("bar this.rooms", this.rooms);
+    MozLoopService.log.debug("bar this.rooms.has(roomToken)", this.rooms.has(roomToken));
+    if (this.rooms.has(roomToken)) {
+      return this.rooms.get(roomToken).participants.length;
+    }
+    // no room, log error and send back 0 to indicate none in room
+    throw new Error("No room found in current session.");
```

Comment:
I think you either want to change `_maybeShowBrowserSharingInfoBar` to check that the room token is valid before passing it to here and/or add a try catch, or you want to change this to return 0 and maybe log an error.